### PR TITLE
changed all alerts to antD alerts

### PR DESCRIPTION
**Pull request by natepace**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by Poonchy — CHANGES_REQUESTED** [review]
Just merge the imports. The rest of the code works fine locally.

**Review by Poonchy — APPROVED** [review]

File: .env.sample [hunk 1]
```diff
@@ -1,3 +0,0 @@
-REACT_APP_CLIENT_ID=getThisFromCredentialsDoc
```
Comment (taylorfriesen6) [inline .env.sample:1]: Do not delete .env.sample

**Comment by taylorfriesen6** [comment]
Other than that, everything else seems good to me, and works on my local machine.

File: .env.sample (hunk 1 above)
Comment (nicholasbus) [inline .env.sample:1]: remove this file from the PR please

**Review by nicholasbus — CHANGES_REQUESTED** [review]
Everything is good, just remove the `.env.sample` deletion from the PR

**Review by taylorfriesen6 — APPROVED** [review]

**Pull request merged**